### PR TITLE
Pull request draft

### PR DESCRIPTION
**Pull request by Kamillkaze**

### CODE REVIEW

File: src/main/java/pl/todolist/ToDoItemController.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class ToDoItemController {
+
+    private final ToDoItemService toDoItemService;
+
+    private final ObjectMapper objectMapper;
```
Comment (Kamillkaze) [inline src/main/java/pl/todolist/ToDoItemController.java:17]: That dependency is not used at all. Please remove it

File: src/main/java/pl/todolist/ToDoItemController.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    public ToDoItemController(ToDoItemService toDoItemService, ObjectMapper objectMapper) {
+        this.toDoItemService = toDoItemService;
+        this.objectMapper = objectMapper;
+    }
+
+    @GetMapping("/todoitems")
+    public ResponseEntity<List<ToDoItemDto>> getToDoItems() throws JsonProcessingException {
```
Comment (Kamillkaze) [inline src/main/java/pl/todolist/ToDoItemController.java:25]: `throws JsonProcessingException` is not needed anymore

File: src/main/java/pl/todolist/ToDoItemController.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
+package pl.todolist;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
```
Comment (Kamillkaze) [inline src/main/java/pl/todolist/ToDoItemController.java:8]: Please optimize imports after other comments are fixed

Comment (KamilkazeCode) [inline src/main/java/pl/todolist/ToDoItemController.java:8]: Please optimize imports

File: src/main/java/pl/todolist/ToDoItemController.java (hunk 1 above)
Comment (KamilkazeCode) [inline src/main/java/pl/todolist/ToDoItemController.java:17]: Not needed

**Review by KamilkazeCode — CHANGES_REQUESTED** [review]

**Review by KamilkazeCode — APPROVED** [review]

**Pull request merged**